packages/engine-test-utils/src/__tests__/pods-core/v2/AirtableExportPodV2.spec.ts

```diff
@@ -629,3 +634,118 @@ describe("GIVEN export note with date ", () => {
     });
   });
 });
+
+describe("GIVEN note has pods namespace in frontmatter", () => {
+  describe("WHEN export note with linked record", () => {
+    let resp: AirtableExportReturnType;
+
+    const setupTest = setupTestFactoryForNote({
+      fname: "proj.beta",
+      srcFieldMapping: {
+        Tasks: {
+          type: "linkedRecord",
+          to: "links",
+          podId: "dendron.task",
+          filter: "task.*",
+        },
+      },
+    });
+
+    describe("AND linked note does not have airtable id", () => {
+      test("THEN show error message", async () => {
+        const preSetupHook = async (opts: WorkspaceOpts) => {
+          await TestEngineUtils.createNoteByFname({
+            fname: "task.alpha",
+            body: "This is a task",
+            custom: {},
+            ...opts,
+          });
+          await TestEngineUtils.createNoteByFname({
+            fname: "proj.beta",
+            body: "[[task.alpha]]",
+            custom: {},
+            ...opts,
+          });
+        };
+        const resp = await setupTest(preSetupHook);
+        await checkString(
+          resp.error!.message,
+          "The following notes are missing airtable ids: dendron://vault1/task.alpha (task.alpha)"
+        );
+      });
+    });
+
+    describe("AND linked note has airtable id", () => {
```

Comment:
can we also test the case where the linked note is associated with multiple records? linking to the note should get the record from the matching podId